packages/dropbox/package.js

```diff
@@ -0,0 +1,29 @@
+Package.describe({
+  name: 'cfs:dropbox',
+  version: '0.0.1',
+  summary: 'Dropbox storage adapter for CollectionFS',
+  git: 'https://github.com/CollectionFS/Meteor-cfs-dropbox.git',
```

Comment:
can we do something like : `https://github.com/CollectionFS/Meteor-CollectionFS.git/packages/dropbox`? will atmosphere support this pattern @tmeasday?
In short we have all core packages in one repo - in an attempt to make the project easier to contribute to and have an overview over. All the packages are placed in a Meteor QA app for development, making it easier to make contributions.